From e1b8dad08b34568522741585c7a03f31e6a81a3b Mon Sep 17 00:00:00 2001
From: cchoux <cosmo.chou@quantatw.com>
Date: Sat, 02 Mar 2024 19:32:15 +0800
Subject: [PATCH] PSUSensor: add sbrmi support

Add device sbrmi for power reading support.

Test result:
- check sensor creation
```
root@bmc:~# busctl tree xyz.openbmc_project.PSUSensor |grep SBRMI
    |   `- /xyz/openbmc_project/State/Decorator/SBRMI_OperationalStatus
        `- /xyz/openbmc_project/sensors/power/SBRMI_Output_Power
```

Change-Id: I4a86071e4bf01ccd473441d395221f565bb15519
Signed-off-by: Cosmo Chou <cosmo.chou@quantatw.com>
---

diff --git a/src/PSUSensorMain.cpp b/src/PSUSensorMain.cpp
index d6bc84c..89ae913 100644
--- a/src/PSUSensorMain.cpp
+++ b/src/PSUSensorMain.cpp
@@ -100,6 +100,7 @@
     {"RAA229001", I2CDeviceType{"raa229001", true}},
     {"RAA229004", I2CDeviceType{"raa229004", true}},
     {"RAA229126", I2CDeviceType{"raa229126", true}},
+    {"SBRMI", I2CDeviceType{"sbrmi", true}},
     {"TDA38640", I2CDeviceType{"tda38640", true}},
     {"TPS53679", I2CDeviceType{"tps53679", true}},
     {"TPS546D24", I2CDeviceType{"tps546d24", true}},
